src/modules/input.js

```diff
@@ -0,0 +1,89 @@
+let todos = [];
```

Comment:
I would recommend using const instead of let when the value of a variable is not intended to change after its initial assignment.